prob/torus_cbc/build.py

```diff
@@ -211,8 +232,7 @@
 # use selection instead
 Rout_rad = ENTROPY*ceil(Rmax) # not safe to use 3x
 Rout_vis = Rout
-GAMMA = 13./9.
-KAPPA = 1.e-3
+GAMMA = 4./3.
```

Comment:
This should be if-cased on whether you're radiation or gas pressure dominated. `4./3.` is appropriate for radiation pressure, not gas pressure.